crypto/fipsmodule/ec/p521.c

```diff
@@ -220,7 +216,13 @@ static void p521_felem_cmovznz(p521_limb_t out[P521_NLIMBS],
 
 // NOTE: the input and output are in little-endian representation.
 static void p521_from_generic(p521_felem out, const EC_FELEM *in) {
+#ifdef OPENSSL_BIG_ENDIAN
+  uint8_t tmp[66];
```

Comment:
can we add `#define P521_FELEM_BYTES (66)` and use the macro instead of constants here and in `p521_from_scalar`?